index.js

```diff
@@ -78,6 +78,7 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
 
 function shouldIncludeEvent(eventType) {
     const isAuthorAlsoTheOwner = eventType.author_association === 'OWNER';
+
```

Comment:
@IngridGdesigns I thought you might find this interesting - Remember your PR was adding this empty line right here and I asked you to take it out so you did? Looks like (based on kevin's commit comment) that space was something the linter did. And now I remember that the linter puts a line before a return statement. I still think it looks stupid, but so it goes.